src/Stripe/Services/Invoices/StripeInvoiceService.cs

```diff
@@ -63,10 +64,11 @@ public virtual IEnumerable<StripeInvoice> List(StripeInvoiceListOptions listOpti
             return Mapper<StripeInvoice>.MapCollectionFromJson(response);
         }
 
-        public virtual StripeInvoice Create(string customerId)
+        public virtual StripeInvoice Create(string customerId, StripeInvoiceCreateOptions createOptions = null)
```

Comment:
Yes i agree, while declaring the createOptions parameter null does make i backwards compatibility for all server side implementations, client side implementations still read and require that optional parameter. So basically this change will break every JS-based implementation.

As far as web goes.